### ISSUE TITLE
OperatorWrap with token ASSIGN too strict for annotations

### ISSUE THREAD

**Issue by robtimus** [checkstyle/checkstyle#10945]
I have read check documentation: https://checkstyle.sourceforge.io/config_whitespace.html#OperatorWrap
I have downloaded the latest checkstyle from: https://checkstyle.org/cmdline.html#Download_and_Run
I have executed the cli and showed it below, as cli describes the problem better than 1,000 words

```bash
/var/tmp $ javac OperatorWrapExample.java

/var/tmp $ cat config.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
    <module name="OperatorWrap">
      <property name="tokens" value="ASSIGN"/>
    </module>
  </module>
</module>

/var/tmp $ cat OperatorWrapExample.java
@Annotation(example = {
        "foo",
        "bar"
})
public class OperatorWrapExample {

    public String[] array = {
            "foo",
            "bar"
    };
}

@interface Annotation {
    String[] example();
}

/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-9.1-all.jar -c config.xml OperatorWrapExample.java
Starting audit...
[ERROR] I:\_tmp\_checkstyle\OperatorWrapExample.java:1:21: '=' should be on a new line. [OperatorWrap]
Audit done.
Checkstyle ends with 1 errors.
```

---

In the above example, the array field allows the first element to be on a new line. If the same is done for an annotation, Checkstyle complains. The only way to "fix" this is to put the first element on the same line as the `= {`. I expect the same behaviour in annotations as in other code.

**Comment by romani** [comment]
There should be no violations on such code. Behavior should be consistent with array for field.
Wrap is for array body, not for `=`.

**Comment by strkkk** [comment]
fix is merged
